packages/chakra-ui/src/SelectWidget/SelectWidget.tsx

```diff
@@ -70,7 +71,7 @@ export default function SelectWidget<
   )
     ? enumOptions.map((option: EnumOptionsType<S>, index: number) => {
         const { value, label } = option;
-        _valueLabelMap[index] = label;
+        _valueLabelMap[index] = label || String(value);
```

Comment:
If no label, fall-back to the stringified value (same as the code below, but better here)